contentcuration/contentcuration/serializers.py

```diff
@@ -357,10 +358,16 @@ class SimplifiedContentNodeSerializer(BulkSerializerMixin, serializers.ModelSeri
         resource_count=ResourceCount(),
         coach_count=CoachCount(),
     )
+    ancestor_query = Metadata(
+        ancestors=AncestorSubquery()
+    )
 
     def retrieve_metadata(self, node):
         return self.metadata_query.get(node.pk)
 
+    def get_node_ancestors(self, node):
+        return filter(lambda a: a, self.ancestor_query.get(node.pk).get('ancestors', []))
```

Comment:
Not as big of a fan of this, but wanted to get this in quickly and I couldn't find how to remove null values using the django expressions

Reply:
Did you try a `Coalesce`? The output field needs to be set to `output_field=JSONField()`, like https://github.com/learningequality/studio/pull/1728/files#diff-aeed3c58c550c180a1a4935742877d8eR151-R159

Reply:
Hmm doesn't seem to be working still

Reply:
I did this in `vue-refactor` to create a `NotNullArrayAgg` that excludes null values and returns an empty list by default:
https://github.com/learningequality/studio/blob/vue-refactor/contentcuration/contentcuration/viewsets/common.py#L8

Reply:
Looking at the code again, I should probably be coercing the filter to a list.